src/TodoListItem.js

```diff
@@ -12,14 +15,33 @@ class TodoListItem extends React.Component {
   }
 
   handleChange = () => {
```

Comment:
Can you remove all the console.logs? They make it hard for me so what is most meaningful in the code. 😄